api-contract/contract.yaml

```diff
@@ -461,7 +598,26 @@ components:
           type: string
         imageProfile:
           type: string
-          format: binary     
+          format: binary   
+    Friendship:
+      type: object
+      properties:
+        username:
+          type: string 
+    FriendList:
+      type: object
+      properties:
+        friendsNames:
+          type: string
+        numFriends:
+          type: integer
+    FriendGroup:
+      type: object
+      properties:
+        friendsNames:
+          type: string
+        groupName:
+          type: string
```

Comment:
This needs to actually return a list of friends. A list is the same as an array, I would expect the type to be 'array' and the schema to be the same as a single user. For example like in here: https://swagger.io/specification/#path-item-object-example